uplink/config.go

```diff
@@ -50,10 +49,7 @@ type EncryptionConfig struct {
 // ClientConfig is a configuration struct for the uplink that controls how
 // to talk to the rest of the network.
 type ClientConfig struct {
-	// TODO(jt): these should probably be the same
-	OverlayAddr   string `help:"Address to contact overlay server through"`
-	PointerDBAddr string `help:"Address to contact pointerdb server through"`
-
+	SatelliteAddr string      `help:"Address to contact the satellite"`
```

Comment:
There are multiple places that reference `client.overlay-addr` and `client.pointer-db-addr`.

Reply:
They should become `client.satellite-addr`, right?

Reply:
yes, unless they were setup from satellite addr

Reply:
Done.